Scripts/script-CommonServer.yml

```diff
@@ -98,15 +98,15 @@ script: |-
       }
 
       //in case of headers was not provided (backward compatibility)
-      if (t && (!(headers instanceof Array && headers.length > 0) || !headers)){
+      if ((!headers || !headers instanceof Array) && t && t.length > 0){
```

Comment:
Do you think that if the caller gave empty array - the function should override this?
(since `[]` is not falsly - i think an empty table should be returned..)

I think it is not the same as giving `null`/`undefined` - but if you think that it should I will fix it of course